app/helpers/application_helper.rb

```diff
@@ -24,4 +24,24 @@ def show_user_name_by_id(id)
     User.find(id).email
   end
 
+  def program_config_data(program)
```

Comment:
I am not clear on what this method is doing.

Reply:
I added it in the previous pull request when the users table did not have a name yet. This is why I left it with email only: User.find(id).email.
I changed the method to display the name and email (User.find(id).display_name_email). 

The method will be used to display the updated_by fields. (Copied from the Art Survey). It's not used anywhere yet because Maria and I could not decide where to display updated_by and other 'history' fields. Program pages are very busy, so we decided to add some fields now and see where to put the rest.

Reply:
```suggestion
  def program_config_data(program)
```

Reply:
I am still not clear what is happening here. Is the idea to present different form fields based on the different configs?